cmake/configure/configure_10_taskflow.cmake

```diff
@@ -30,36 +30,36 @@ macro(feature_taskflow_find_external var)
     set(${var} TRUE)
   endif()
 
-  if(TASKFLOW_VERSION VERSION_LESS "2.4")
+  if(TASKFLOW_VERSION VERSION_LESS "3.7")
     # Clear the previously determined version numbers to avoid confusion
     set(TASKFLOW_VERSION "bundled")
     set(TASKFLOW_VERSION_MAJOR "")
     set(TASKFLOW_VERSION_MINOR "")
 
     message(STATUS
-      "The externally provided Taskflow library is older than version 2.4, "
+      "The externally provided Taskflow library is older than version 3.7, "
       "which cannot be used with deal.II."
       )
     set(TASKFLOW_ADDITIONAL_ERROR_STRING
       "The externally provided Taskflow library is older than version\n"
-      "2.4, which is the oldest version compatible with deal.II."
+      "3.7, which is the oldest version compatible with deal.II."
       )
     set(${var} FALSE)
   endif()
 
 
-  if(NOT TASKFLOW_VERSION VERSION_LESS "3.0" AND NOT DEAL_II_HAVE_CXX17)
+  if(NOT DEAL_II_HAVE_CXX17)
     # Clear the previously determined version numbers to avoid confusion
     set(TASKFLOW_VERSION "bundled")
     set(TASKFLOW_VERSION_MAJOR "")
     set(TASKFLOW_VERSION_MINOR "")
```

Comment:
This can't be right -- we *always* have C++17 enabled now. I think you can just remove the whole block.